apps/emqx_authz/src/emqx_authz.app.src

```diff
@@ -0,0 +1,17 @@
+{application, emqx_authz,
```

Comment:
It's hard to understand the word `authz`, if it is for ACL, then we could call the application as `emqx_acl`.

Reply:
@tigercl